gm4_soul_glass/data/gm4_soul_glass/functions/effect/level_4_strong.mcfunction

```diff
@@ -5,4 +5,4 @@ execute if score @s gm4_sg_primary matches 1 run effect give @a[dx=100,dy=356,dz
 execute if score @s gm4_sg_primary matches 3 run effect give @a[dx=100,dy=356,dz=100] minecraft:mining_fatigue 17 1
 execute if score @s gm4_sg_primary matches 5 run effect give @a[dx=100,dy=356,dz=100] minecraft:weakness 17 1
 execute if score @s gm4_sg_primary matches 8 run effect give @a[dx=100,dy=356,dz=100] minecraft:slow_falling 17 1
-execute if score @s gm4_sg_primary matches 11 run effect give @a[dx=100,dy=356,dz=100] minecraft:glowing 17 1
+execute if score @s gm4_sg_primary matches 11 run effect give @a[dx=100,dy=356,dz=100] minecraft:glowing 17 0
```

Comment:
I was thinking because higher levels of Glowing does not change anything, it would make more sense to cap it at 1, but we might need others' opinions on this.